### DISCUSSION TITLE
OrderSearch CustomParameters

### DISCUSSION THREAD

**Discussion by SnusRasmus** [FortnoxAB/csharp-api-sdk#287] [Q&A]
I just want one order based on the "Your Order Number"

With the code below i get "invalid parameter"
Anyone know what I do wrong ?

````
var dictionary = new Dictionary<string, string>();
dictionary.Add("YourOrderNumber", "123");
var orders = await fortnoxConnection.GetOrdersAsync(new Fortnox.SDK.Search.OrderSearch() { CustomParameters = dictionary });
````

**Comment by adamelfstrom** [comment]
It does not appear that you can search for orders using YourOrderNumber.

If you take a look at the OrderSearch class you can see the parameters that are implemented in the SDK currently. These are based on the list seen here https://apps.fortnox.se/apidocs#tag/OrdersResource/operation/list_OrdersResource in the "Retrieve a list of orders" query parameter list.
